Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -641,21 +683,21 @@ class StoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
         await waitForNewPurchaseDate()
 
         // 4. Expire subscription
-        try await self.expireSubscription(entitlement)
+//        try await self.expireSubscription(entitlement)
```

Comment:
Is commenting out the latter part of this test intentional?

Reply:
Nope! I noticed this a bit ago and am fixing 😛